base/ubi8/entrypoint.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
```

Comment:
We are not using EPL for this project but [Apache 2.0](https://github.com/devfile/developer-images/blob/main/LICENSE)

Reply:
Fixed